app/src/controllers/project/sagas.js

```diff
@@ -140,6 +142,7 @@ function* deleteDefectSubType({ payload: subType }) {
         type: NOTIFICATION_TYPES.SUCCESS,
       }),
     );
+    yield put(hideModalAction());
   } catch (error) {
     yield put(showDefaultErrorNotification(error));
   }
```

Comment:
Please rename all places with `sub` word to transfer a defect `subType` to `defectType`